campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md

```diff
@@ -139,7 +139,7 @@ Content-Type: application/xml
 Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/imports?page=1&size=2
 
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<LongCodeImportCampaignsResponse>
+<LongCodeImportCampaignResponse>
```

Comment:
I believe this one (and the following error one) will actually be LongCodeImportCampaignsResponse as written, since it's returning multiple.